microsoft-edge/webdriver-chromium/capabilities-edge-options.md

```diff
@@ -63,6 +63,7 @@ Most Microsoft Edge-specific capabilities are exposed through the `EdgeOptions`
 | `wdpAddress` | string | An address of a Windows Device Portal server to connect to, in the form of `hostname/ip:port`, for example  `127.0.0.1:50080`.  For more information, see [Remote Debugging - Windows 10 devices](../devtools-guide-chromium/remote-debugging/windows.md). |
 | `wdpPassword` | string | Optional password to use when connecting to a Windows Device Portal server.  Required if the server has authentication enabled. |
 | `wdpUsername` | string | Optional user name to use when connecting to a Windows Device Portal server.  Required if the server has authentication enabled. |
+| `wdpProcessId` | integer | Required processId to use if attaching to a running WebView2 UWP app, for example `36590`. This information can be found in `browserProcessId` on `http://<Device Portal URL>/msedge`. |

```

Comment:
```suggestion
| `wdpProcessId` | integer | Required `processId` to use if attaching to a running WebView2 UWP app, for example `36590`. This information can be found in `browserProcessId` on `http://<Device Portal URL>/msedge`. |
```
use that change if `processId` is a specific thing in the API.  
else maybe use English words:
The required process ID to use if attaching to a running WebView2 UWP app,

Reply:
pretty bad GitHub UI comment-entry bugs today, trying again...